guides/players/radios.md

```diff
@@ -0,0 +1,49 @@
+# ACRE Radios
+Need a brief here (This page will go over the basic operation of ACRE radios for Synixe etc)
+## Controls
+Pressing CapsLock will start transmitting through the radio (indicated by a click) until you let go of CapsLock.
+
+When interacting with any of the ACRE radios there are two different ways to bring up their interface to change the Channel Frequency:
+* Double clicking the radio within your inventory 
+* Self interacting via ACE 
+
+<hr/>
+
+# AN/PRC-343 Usage
+Need a brief here
+### Volume Control (Combine)
+Left clicking the volume knob increases the 343's volume by 10% to a maximum of 100% whilst right clicking will decrease by 10% to a minimum of 20% volume.
+
+### Channel & Block Control (Seek better formatting)
+To change the channel you left click or right click (increase/decrease by 1) the Channel knob to cycle through the 16 channels.
+![alt text](https://cdn.discordapp.com/attachments/386271418305871873/566169026359853069/107410_20190412192712_1.png)

```

Comment:
Images should go under the brief, before any specifics. This way people have seen the image before the start reading what buttons do.

Reply:
No.